packages/scripts.nix

```diff
@@ -497,17 +508,10 @@
       busybox
       kubectl
       dasel
+      cleanup-images
```

Comment:
You can remove this from the let binding and refer to it as scripts.cleanup-images